compiler/src/dmd/parse.d

```diff
@@ -4857,6 +4857,10 @@ class Parser(AST, Lexer = dmd.lexer.Lexer) : Lexer
                     token.value == TOK.identifier && peekNext() == TOK.goesTo ||
                     token.value == TOK.ref_ && peekNext() == TOK.leftParenthesis &&
                         skipAttributes(peekPastParen(peek(&token)), &tk) &&
+                        (tk.value == TOK.goesTo || tk.value == TOK.leftCurly) ||
+                    token.value == TOK.auto_ && peekNext() == TOK.ref_ &&
```

Comment:
Ident issue

Reply:
The non-standard indentation here is intentional, and present in the original code. Each indented block is part of the same top-level `||` term as its non-indented "header" line, and each corresponds to a different `token.value`.